lib/nimble_template/addons/variants/phoenix/web/core_js.ex

```diff
@@ -24,7 +24,7 @@ defmodule NimbleTemplate.Addons.Phoenix.Web.CoreJS do
       """,
       """
         "dependencies": {
-          "core-js": "^3.7.0",
+          "core-js": "^3.7.0"
```

Comment:
please consider using the latest version https://www.npmjs.com/package/core-js/v/3.21.1
```suggestion
          "core-js": "^3.21.1"
```

Reply:
It's not a part of this PR I think @longnd, we can create an issue to upgrade all Node dependency version after merge this PR

Reply:
same issue on the upgrade Elixir and Erlang version, update each Node dependency might have some affect (the new version doesn't work) so prefer to do that on a separate PR @longnd